### ISSUE TITLE
Doesn't show all emails

### ISSUE THREAD

**Issue by steveja42** [eneam/mboxviewer#52]
I exported all my gmail to a 7.9GB .mbox file.  
I opened it in mboxviewer, and properties says Mail Count:8757, and some of the gmail labels are obviously missing emails.
In Thunderbird the same .mbox file shows 82,535 total messages.

**Comment by zigm** [comment]
Obviously there seems to be a problem somewhere. Let me know what version you are running. At this point please re-parse the mbox file and let me know if that helps. Right click on the mbox file under Tree view and select "Refresh Index File".

Also, are your mbx file located on the local or network drive?

**Comment by steveja42** [comment]
Refreshing the index file fixed it, now shows mail count of 82535.
Version 1.03.40 on a  local drive. I had cancelled the indexing before because I was trying to change the data folder.

**Comment by zigm** [comment]
I agree that Cancel action may not be clear. I may consider to remove partial index file created by Cancel or display Message that Refresh is required.

**Comment by zigm** [comment]
Just curious. You are obviously familiar with Thunderbird. It is quite powerful mail client. Do you use both Thunderbird and mboxviewer for mail viewing? What are the pros and cons of both tools from your point of view.

**Comment by steveja42** [comment]
I'm not a Thunderbird expert, but mboxviewer seems to be much better for
working with .mbox files because it shows the gmail labels and
thunderbird doesn't.

**Comment by zigm** [comment]
Thanks. User feedback helps to make MBox Viewer more user friendly.

**Comment by zigm** [comment]
New release v1.0.3.41 will notify users on possible actions to take when mbox file parsing is Cancelled.

**Comment by pradeepkatiyar007** [comment]
In this case, you can also use the [Shoviv MBOX Viewer](https://www.shoviv.com/mbox-viewer.html) tool, which is feature rich.

Read more: [Free MBOX Viewer](https://apps.microsoft.com/detail/xp9cfpcvq9xpb0)
